src/osemgrep/cli_scan/Scan_CLI.ml

```diff
@@ -335,6 +339,24 @@ let o_force_color : bool Term.t =
 a TTY; defaults to using the TTY status
 |}
 
+let o_max_chars_per_line : int Term.t =
```

Comment:
you can remove the code in TOPORT/commands/scan.py corresponding to those ported flags.

Reply:
done in 74e80af3d